### PR TITLE
Win32 Tabs control updates

### PR DESCRIPTION
**Pull request by sriya112**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [x] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Addressing Patrick's comments from our previous [PR](https://github.com/microsoft/fluentui-react-native/pull/836) for the Tabs control on Win32:
- improved comments by adding a period at the end of each comment
- made props const on the tester app
- updated SVGIconProps

Changed the selected hovered behavior to match the Figma design (the indicator expands when you hover over a selected TabsItem)

Changed the colors to satisfy high contrast requirements and match the Figma design better

### Verification

(how the change was tested, including both manual and automated tests)

Before the selected hovered behavior was updated, the selected hovered behavior was the same as the selected behavior. But now, the selected hovered behavior makes the indicator grow:

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [x] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Comment by samuelfreiberg** [comment]
Can you please be more specific with what this PR is addressing?

Along with "Addressing Patrick's comments from our previous PR for the Tabs control on Win32 (improving commenting, tester app examples)", it would be helpful to either state those comments, or explain everything your tackling in this PR.

Ie. What does "improving commenting" mean? Were there other comments you're addressing in this PR too?

File: packages/components/Tabs/src/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
+/** @jsx withSlots */
+import * as React from 'react';
```
Comment (samuelfreiberg) [inline packages/components/Tabs/src/Tabs.tsx:2]: I'm a bit confused... It looks like these are all new files, but you added them before, correct?

Perhaps you need to pull/merge?

Comment (sriya112) [reply packages/components/Tabs/src/Tabs.tsx]: yes, it's been updated now!

**Review by samuelfreiberg — APPROVED** [review]

**Comment by samuelfreiberg** [comment]
I'd add a screenshot of the new hover styling and compare it with the old styling

**Review by rurikoaraki — APPROVED** [review]

**Pull request merged**